test/version_solver_test.dart

```diff
@@ -421,10 +421,13 @@ void devDependency() {
         })
       ]).create();
 
-      await expectResolves(error: equalsIgnoringWhitespace('''
-        Because myapp depends on both foo from path foo and foo from path
+      await expectResolves(error: anyOf(equalsIgnoringWhitespace('''
+      Because myapp depends on both foo from path foo and foo from path
           ../foo, version solving failed.
-      '''));
+      '''), equalsIgnoringWhitespace('''
+        Because myapp depends on both foo from path foo and foo from path
+          ..\\foo, version solving failed.
```

Comment:
note that you should be able to use `Platform.separator` instead of two separate matchers.

Reply:
smart!
Thanks